### ISSUE TITLE
M$ Edge Flatpak Linux (Big Linux base Manjaro) Does not prevent screen sleep/shutdown.

### ISSUE THREAD

**Issue by Serpentario** [MicrosoftEdge/Status#715]
M$ Edge Flatpak Linux (Big Linux Base Manjaro) Não impede o descanso / desligamento de tela.

Even with video running on any type of screen: windowed or fullscreen mode, when the system reaches the given time limit to turn off the screen, turning it off, M$ Edge (Flatpak since AUR doesn't install on BigLinux) simply is ignored by the OS and the screen is turned off, only the audio is kept working.

If you press any key or move the Mouse, the video returns.

In summary, M$ Edge Flatpak does not prevent the system from turning off the monitor with video running.
